ipatests/test_ipaclient/data/test_csrgen/scripts/caIPAserviceCert_openssl.sh

```diff
@@ -29,5 +30,5 @@ DNS = machine.example.com
 subjectAltName = @sec1
 ' > "$CONFIG"
 
-openssl req -new -config "$CONFIG" -out "$CSR" -key $1
-rm "$CONFIG"
+openssl req -new -config "$CONFIG" -out "$CSR" -key "$KEYFILE" "$@"
+rm "$CONFIG"
```

Comment:
The missing newline breaks `test_CSRGenerator.test_caIPAserviceCert_OpenSSL` test case.